types/extra-types.d.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * This file contains the types that are required for compilation of the
+ * Polymer generated type declarations, but which could not themselves be
+ * automatically generated.
+ */
+
+/// <reference path="lib/mixins/property-effects.d.ts" />
+
+/**
```

Comment:
Shouldn't this be a `/*` comment, since it's not a doc comment on the subsequent line?

Reply:
Done.